CHANGELOG.md

```diff
@@ -76,10 +77,12 @@
 
 ### Fixes
 
+- Fixed immediate language updates of application top menu bar ([PR 2813](https://github.com/input-output-hk/daedalus/pull/2813))
```

Comment:
Why adding it in 4.7.0? @mchappell

Reply:
Actually it is correct that this line is added. It was removed previously by merge commits. I don't know what was the reason but maybe this was just an oversight. Anyway it is ok that we are adding it back.

Suspicious commits:
A merge of develop to a branch before merging PR: https://github.com/input-output-hk/daedalus/pull/2805/commits/b7fd5a75dd651fa99ff8f1f6426c44992cba1ff9
PR merge commit: https://github.com/input-output-hk/daedalus/commit/37495fae0668dbee552747c748aecfcfaf0b17df